test/e2e/performanceprofile/functests/0_config/config.go

```diff
@@ -134,6 +135,14 @@ var _ = Describe("[performance][config] Performance configuration", Ordered, fun
 		format.Object(performanceProfile, 2)
 	})
 
+	It("should create SRI-OV devices if they are exist", func() {
+		sriovInfos, err := sriovcluster.DiscoverSriov(testclient.SRIOVClient, sriov.OperatorNamespace)
```

Comment:
from what I undertand, devices related tests will run if the operator is installed on the sriov.OperatorNamespace, and installing the operator is not part of the tests setup. 
this means we need to setup the operator installation in the d/s ci pipelines.

Reply:
Yes. I also know that on cnf-test they are installing the operator before the test starts:
https://github.com/openshift-kni/cnf-features-deploy/blob/master/hack/setup-build-index-image.sh#L62-L66